executor/index_merge_reader.go

```diff
@@ -717,3 +793,65 @@ func (w *indexMergeTableScanWorker) executeTask(ctx context.Context, task *looku
 	}
 	return nil
 }
+
+// IndexMergeRuntimeStat record the indexMerge runtime stat
+type IndexMergeRuntimeStat struct {
+	IndexMergeProcess int64
+	IndexScan         int64
+	TableRowScan      int64
```

Comment:
ditto.
How about using `time.Duration` and suffixing them with `Time`?

Reply:
I don't think it's ok 😂, the IndexScan and TableRowScan is need to time in the go routine, if we don't use int64 and atomic.AddInt64(), maybe it will have race condition😂